fd_linux_1.10.go

```diff
@@ -18,6 +19,16 @@ func (lfd *sysListenFD) accept4(flags int) (int, unix.Sockaddr, error) {
 	return unix.Accept4(lfd.fd, flags)
 }
 
+func (lfd *sysListenFD) setNonblocking(name string) error {
+	if err := unix.SetNonblock(lfd.fd, false); err != nil {
```

Comment:
This should have a comment explaining that Go 1.10 cannot make use of non-blocking I/O. Same with sysConnFD equivalent.